### DISCUSSION TITLE
How to change docker export port

### DISCUSSION THREAD

**Discussion by panw3i** [minio/minio#12831] [Q&A] [answered]
```
docker run \
  -p 9010:9000 \
  -p 9011:9001 \
  -e "MINIO_ROOT_USER=AKIAIOSFODNN7EXAMPLE" \
  -e "MINIO_ROOT_PASSWORD=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" \
  minio/minio server /data --console-address ":9011"

```

`docker run` result 

```
API: http://172.17.0.4:9000  http://127.0.0.1:9000

Console: http://172.17.0.4:9011 http://127.0.0.1:9011

Documentation: https://docs.min.io

```

open  `http://localhost:9010` will redirect to `http://localhost:9010` and  page  show ` This page is not working`

how can I  slove it ?

**Comment by harshavardhana** [comment, marked as answer]
@panw3i please read the docs https://github.com/minio/minio#test-using-minio-console

**Reply by cloudscape-germany** [reply]
Its not saying about API port, also in my tests MinIO insists on Port 9000, no matter what environment flags I use.
